sda/cmd/ingest/ingest.go

```diff
@@ -100,6 +101,24 @@ func main() {
 	var message schema.IngestionTrigger
 
 	go func() {
+		start := time.Now()
+		for i := 0; i <= 10; i++ {
+			h, err := db.ListKeyHashes()
+			if err != nil {
+				log.Errorln(err.Error())
+			}
+			if len(h) != 0 {
+				break
+			}
+
+			time.Sleep(time.Duration(30 * time.Second))
```

Comment:
So as the sleep duration, better to avoid hard coding

Reply:
Sleep is there so as not to put a strain on the DB connection and not spam the log messages.